java-client/session-dagger/src/test/java/io/deephaven/client/impl/TableServiceAsyncTest.java

```diff
@@ -93,8 +93,11 @@ private static List<TableSpec> createLongChain(int numColumns, int numRows) {
             if (i == 0) {
                 longChain.add(TableSpec.empty(numRows).view("I_0=ii"));
             } else {
+                // create a chain using the previous table even though we don't need the data
                 final TableSpec prev = longChain.get(i - 1);
-                longChain.add(prev.updateView("I_" + i + " = 1 + I_" + (i - 1)));
+                // to avoid timeouts due to compilation variance reuse the same expression, including the destination
+                // column which shows up in the generated formula's FormulaEvaluationException
+                longChain.add(prev.updateView("I_1 = 1 + I_0"));
```

Comment:
I'm not sure how this is directly related to #4798; that is _specifically_ around a timeout error. I'd assume in the case of too many open files in the compilation path there will be some _other_ error that causes TableServiceAsyncTest to fail, correct?

Reply:
Everytime it was timed out the server was in the middle of compiling. I did a bunch of instrumentation - there is no real issue with the test, just that the variance of compilation causes it to time out. Repeated local runs are actually harder to time out because the classes have been precompiled.

The change that you are commenting on here, is the actual fix -- run the long chain without repeatedly invoking the compiler. If you prefer, we can split this out from the singleton creation.

Reply:
Ah, makes sense; I never saw that high of variation likely due to the points you bring up.